client/src/pages/ClientSignupPage.js

```diff
@@ -12,10 +21,362 @@ function ClientSignupPage(props) {
         <NavbarComponent links={employeeNavbarLinks} />
       </Row>
       <Row>
-        <h1>ClientSignupPage</h1>
+      <ClientPage {...props} />
+      </Row>
+    </Container>
+  );
+}
+
+
+
+function ClientPage(props) {
+  return (
+    <Container>
+      {" "}
+      <Row>
+        <ClientForm {...props}/>
       </Row>
     </Container>
   );
 }
 
+
+function ClientForm(props) {
+  
+  const [success, setSuccess] = useState(false);
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+  const [errorMessage1, setErrorMessage1] = useState("");
+  const [errorMessage2, setErrorMessage2] = useState("");
+  const [errorMessage3, setErrorMessage3] = useState("");
+  const [errorMessage4, setErrorMessage4] = useState("");
+  const [errorMessage5, setErrorMessage5] = useState("");
+  const [errorMessage6, setErrorMessage6] = useState("");
+  const [errorMessage7, setErrorMessage7] = useState("");
+  const [errorMessage8, setErrorMessage8] = useState("");
+  
+  
+  
+
+  
+  const [client, setClient] = useState({ firstName: "", lastName: "",phoneNumber: "", email: " ", address:"", number:"", city:"", postCode:"" });
+  const cancelErrors=()=>{
+    setErrorMessage("");
```

Comment:
I don't get why you've created all those states for error messages. If you show the alert at the top of the form and not one alert for each field I think you can define only one state. Just check it and in case change this